src/main/java/com/hortonworks/spark/sql/hive/llap/HiveWarehouseDataWriter.java

```diff
@@ -0,0 +1,69 @@
+package com.hortonworks.spark.sql.hive.llap;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.BytesWritable;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.mapred.TaskAttemptContext;
+import org.apache.hadoop.mapred.TaskAttemptContextImpl;
```

Comment:
because Spark dropped Hadoop 1 support as of Spark 2.0.0 (https://issues.apache.org/jira/browse/SPARK-11807) and I thought we could use Mapreduce 2 API if I am not mistaken.

Reply:
@HyukjinKwon 
Good point. I switched it now.